app/views/backgrounds/_form.html.erb

```diff
@@ -1,31 +0,0 @@
-<%= form_with(model: background, local: true) do |form| %>
```

Comment:
Do you need to delete the previous background create actions? 
As we discuss, we should keep this and make a revamped dedicated page for backgrounds later.

Reply:
This page has become unusable because it loads all samples that are in the database onto the page. Therefore the page is so slow and unwieldy that we've been unable to use it for background creation.
The new feature in this PR replicates the same functionality (select samples, enter name, create background) but in a way that is actually usable

Reply:
Yeah, in any case this does not use react as all other components.